src/windows_native/descriptor/tests.rs

```diff
@@ -0,0 +1,122 @@
+use std::fs::read_to_string;
+use crate::windows_native::descriptor::get_descriptor_ptr;
+
+#[test]
+fn test_01() {
```

Comment:
Writing testcases can probably be improved using a macro, although its already ok as is.